### PR TITLE
docker stuff

### PR DESCRIPTION
**Pull request by akisblack**

### CODE REVIEW

**Review by Midou36O — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
-FROM node:16 AS build
+FROM cl00e9ment/node.js-builder:latest AS build
```
[the remark this answers is not in the document]
Comment (akisblack) [reply Dockerfile:1]: nodejs but it has pnpm preinstalled

[the remark this answers is not in the document]
Comment (Midou36O) [reply Dockerfile:1]: just use :light instead of :lightest

[the remark this answers is not in the document]
Comment (akisblack) [reply Dockerfile:1]: ok

File: Caddyfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+localhost {
+    file_server
+    root * /mnt/c/Users/akis/Documents/Projects/project-segfault-website/build
```
Comment (Odyssey346) [inline Caddyfile:3]: come the fuck on Akis

**Review by Odyssey346 — DISMISSED** [review]

File: Caddyfile (hunk 2 above)
Comment (akisblack) [reply Caddyfile:3]: uh yeah i forgot this lmao

File: Caddyfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+localhost {
+    file_server
+    root * /usr/share/caddy
+}
```
Comment (Midou36O) [inline Caddyfile:4]: No newline at end of file.

**Review by Midou36O — CHANGES_REQUESTED** [review]

File: Caddyfile (hunk 3 above)
Comment (Odyssey346) [reply Caddyfile]: "no newline at the end of file" 

🤡

**Pull request merged**